test/widgets/emoji_reaction_test.dart

```diff
@@ -471,6 +469,87 @@ void main() {
 
       debugNetworkImageHttpClientProvider = null;
     });
+
+    group('handle view paddings', () {
+      const screenHeight = 400.0;
+
+      late double scrollViewTop;
+      late double scrollViewBottom;
+      final scrollViewFinder = findInPicker(find.bySubtype<ScrollView>());
+
+      late Widget firstOption;
+      late Widget lastOption;
+      double getScrollChildrenTop(WidgetTester tester) =>
+        tester.getTopLeft(find.byWidget(firstOption)).dy;
+      double getScrollChildrenBottom(WidgetTester tester) =>
+        tester.getBottomLeft(find.byWidget(lastOption)).dy;
```

Comment:
The name, return type, and implementation don't seem to fit with each other; I can say more if helpful.

Reply:
Yeah, I kind of struggled naming these helpers. I meant to express that this fetches y-coord of the top-most/bottom-most edge of the scrollable content (scroll items/scroll list?). 

Are `double getScrollElementTopEdgeDy` and `double getScrollElementBottomEdgeDy` better?